### PR TITLE
add snapshot time/loss stat in train.py

### PR DESCRIPTION
**Pull request by heavengate**
log average loss and iter time of snapshot like
```
2019-07-03 08:37:12,689-INFO: Save model to output/mask_rcnn_r50_1x/40.
2019-07-03 08:37:13,387-INFO: Snapshot 40: average loss: 1.968849, average time: 0.704
```

### CODE REVIEW

File: PaddleCV/PaddleDetection/tools/train.py [hunk 1]
```diff
@@ -163,10 +165,20 @@ def main():
...
             it, np.mean(outs[-1]), logs, end_time - start_time)
         logger.info(strs)
+        snapshot_loss += stats['loss']
+        snapshot_time += end_time - start_time
 
         if it > 0 and it % cfg.snapshot_iter == 0:
             checkpoint.save(exe, train_prog, os.path.join(save_dir, str(it)))
 
+            # log out snapshot stat
+            snapshot_loss /= cfg.snapshot_iter
```
Comment (willthefrog) [inline PaddleCV/PaddleDetection/tools/train.py:175]: need to take `start_iter` into account? may be less than `snapshot_iter`

Comment (qingqing01) [reply PaddleCV/PaddleDetection/tools/train.py:175]: 实际上，我感觉不用输出也行？
1. 可以通过输出log的间隔计算。
2. 可以ll 输出的模型，查看保存的间隔，即可。
